docs/src/icons/SvgBaseUiLogo.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import RootSvg, { RootSvgProps } from 'docs/src/icons/RootSvg';
+
+export default function SvgBaseUiLogo(props: RootSvgProps) {
+  return (
+    <RootSvg
+      xmlns="http://www.w3.org/2000/svg"
+      width={24}
+      height={24}
+      viewBox="0 0 24 24"
+      fill="none"
+      {...props}
+    >
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        fill="#007FFF"
+        d="M2 0a9 9 0 0 1 9 9v3H2V0Zm10 24c5.523 0 10-4.477 10-10H2c0 5.523 4.477 10 10 10Zm9.315-15.444A9 9 0 0 1 22 12h-9V3a9 9 0 0 1 8.315 5.556Z"
+      />
+    </RootSvg>
+  );
+}
+
+export const BaseSvgLogoString = `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
By convention, capitalized named are reserved for react components. This const is a string. Same for the other svg strings

```suggestion
export const baseSvgLogoString = `<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```